src/to_wiki.js

```diff
@@ -74,18 +76,66 @@ class WikiTextSerializerState {
    }
 
    // Apply marks to inline text.
+   // Find lengths of each continuous mark use and store in Map. (This can't be
+   // done simultaneously with the next step, because we need to know the full
```

Comment:
The comment confused me a little bit. I think I'd find this a bit clearer:

```js
// Do a first pass over the nodes to get the length of each continuous mark.
// Save this information for the second pass.
// (rest of your comment here)
```

Up to you, I might just not be familiar enough with prosemirror to understand the terms.